src/desktop/apps/order2/server.js

```diff
@@ -44,6 +44,7 @@ app.get('/order2/:orderID*', async (req, res, next) => {
         styledComponents: true,
       },
       blocks: {
+        head: () => headTags,
```

Comment:
Looking through https://github.com/tizmagik/react-head this seems like a solid choice, but one thing i'm wondering about is the ability to naturally merge `headTags` into other hand-written solutions [such as this](https://github.com/artsy/force/blob/master/src/desktop/apps/artist2/components/Meta.js) which is [used here](https://github.com/artsy/force/blob/master/src/desktop/apps/artist2/server.js#L66).

I imagine it's as simple as passing `headTags` into `<Meta>` and then interpolating via `{headTags}`?

Reply:
```jsx
{
  head: () => 
  <> 
    ${headTags}
    <Meta/>
  </>
}
```

That would probably work. I think this library would give us a bit of refactoring opportunities there though.

Reply:
👍